zkvm/spec/ZkVM.md

```diff
@@ -737,12 +737,12 @@ Aggregated Signature is encoded as a 64-byte [data](#data-type).
 
 The protocol is the following:
 
-1. Prover and verifier obtain a [transcript](#transcript) `T` defined by the context in which the signature is used (see [`signtx`](#signtx), [`delegate`](#delegate)). The transcript is assumed to be already bound to the _message_ being signed.
+1. Prover and verifier obtain a [transcript](#transcript) `T` that is assumed to be already bound to the _message_ being signed (see [`signtx`](#signtx) and [transaction signature](#transaction-signature)).
 2. Commit the count `n` of verification keys as [LE32](#le32):
     ```
     T.commit("n", LE32(n))
     ```
-3. Commit all verification keys `P[i]` in order, one by one:
+3. Commit all verification keys `P[i]` one by one (in order they were collected):
```

Comment:
nit: this phrasing still seems a little unclear to me. Maybe "one by one (in the order they were added in the program)" ?

Reply:
<img width="845" alt="image" src="https://user-images.githubusercontent.com/698/52676566-60e95580-2edf-11e9-9ffc-8cb776b4e242.png">

how about this wording?

Reply:
great!!!!